eng/common/pipelines/templates/archetype-typespec-emitter.yml

```diff
@@ -202,16 +202,19 @@ extends:
                 Write-Host "Using overrides.json to generate emitter-package.json"
                 tsp-client generate-config-files `
                 --package-json '$(buildArtifactsPath)/lock-files/package.json' `
-                --output-dir '$(Build.ArtifactStagingDirectory)' `
-                --emitter-package-json-path  '${{ parameters.EmitterPackageJsonPath }}' `
-                --overrides '$(buildArtifactsPath)/overrides.json'
+                --output-dir '$(Build.SourcesDirectory)' `
+                --emitter-package-json-path "$(Build.SourcesDirectory)/${{ parameters.EmitterPackageJsonPath }}" `
+                --overrides '$(buildArtifactsPath)/overrides.json' `
+                --version '$(emitterVersion)'
```

Comment:
`--version` isn't a parameter to `generate-config-files`. It should be extracted from `--package-json`